linux_os/guide/system/permissions/restrictions/enable_execshield_settings/sysctl_kernel_randomize_va_space.rule

```diff
@@ -1,7 +1,5 @@
 documentation_complete: true
 
-prodtype: rhel6,rhel7,fedora
```

Comment:
Ubuntu: https://askubuntu.com/questions/318315/how-can-i-temporarily-disable-aslr-address-space-layout-randomization and https://linux-audit.com/linux-aslr-and-kernelrandomize_va_space-setting/
Oracle: https://docs.oracle.com/cd/E37670_01/E36387/html/ol_aslr_sec.html
SUSE: https://en.opensuse.org/openSUSE:Security_Features